akka-stream/src/main/boilerplate/akka/japi/tuple/Tuple.scala.template

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.japi.tuple
```

Comment:
hmm, are there no tuple classes available now in the japi package of 2.3.11? @patriknw ?